EasyPost/Models/API/Event.cs

```diff
@@ -33,7 +34,7 @@ internal Event()
         }
     }
 
-    public class EventCollection : Collection
+    public class EventCollection : PaginatedCollection<Event>
```

Comment:
question: I don't love we're changing the return type here for fear of what it may to do user integrations. What is the user-impact of making this change?

Reply:
We're not changing the return type, necessarily, but the underlying base class for all the `XCollection` classes. I don't imagine any user would be doing something along the lines of "I want to get a `Collection` class, but not specifically a `AddressCollection` class"